src/rootfs.rs

```diff
@@ -253,7 +253,8 @@ fn mount_to_container(
     data: &str,
     label: Option<&String>,
 ) -> Result<()> {
-    let typ = m.typ.as_ref().context("no type in mount spec")?;
+    let none = "".to_string();
+    let typ: &str = m.typ.as_ref().unwrap_or(&none);
     let d = if let Some(l) = label {
         if typ != "proc" && typ != "sysfs" {
```

Comment:
How about this code?
```suggestion
    let typ = m.typ.as_deref();
    let d = if let Some(l) = label {
        if typ != Some("proc") && typ != Some("sysfs") {
```